pyinfraformat/core/io.py

```diff
@@ -37,8 +41,11 @@ def from_infraformat(path=None, encoding="auto", extension=None, robust=True):
     use_glob : bool, optional, default False
         path is a glob string
     extension : bool, optional, default None
-    robust : bool, optional, default False
-        If True, enable reading files with ill-defined/illegal lines.
+    errors : 'ignore_lines', 'ignore_holes' or 'raise', default 'ignore_lines'
+        How to handle ill-defined/illegal lines or holes.
+    save_ignored : str, StringIO or False, default False
```

Comment:
```suggestion
    save_ignored : str, StringIO, FileIO or False, default False
```

Might want to check what was the correct name for this